cloudbuild.yaml

```diff
@@ -0,0 +1,98 @@
+substitutions:
+  _REPO: k8s-container-image-promoter
+  _GOLANGCI_LINT_VERSION: v1.13.2
+  _GOPATH: /workspace/go
+
+steps:
+- name: "gcr.io/cloud-builders/git"
+  id: setup
+  entrypoint: "bash"
+  args:
+  - "-c"
```

Comment:
is it usual to write these scripts entirely inline as opposed to writing them as files in the repo and calling them?

Reply:
Depends. In this case this logic is only meant to be used by the CI (no one else), so keeping it all inside this single file isn't so bad.  /shrug